File: crud-clientes/Cliente.php
<?php
class Cliente {
  public $id;
  public $cedula;
  public $nombres;
  public $email;
  public $telefono;
  public $direccion;
  public $estado;

  public function __construct($data = []) {
    $this->id = $data["id"] ?? null;
    $this->cedula = $data["cedula"] ?? "";
    $this->nombres = $data["nombres"] ?? "";
    $this->email = $data["email"] ?? "";
    $this->telefono = $data["telefono"] ?? "";
    $this->direccion = $data["direccion"] ?? "";
    $this->estado = isset($data["estado"]) ? (bool)$data["estado"] : true;
  }

  public function toArray() {
    return [
      "cedula" => $this->cedula,
      "nombres" => $this->nombres,
      "email" => $this->email,
      "telefono" => $this->telefono,
      "direccion" => $this->direccion,
      "estado" => $this->estado
    ];
  }
}
?>

File: Models/Cliente.cs
namespace ApiCobranzas.Models
{
    public class Cliente
    {
        public int Id { get; set; }
        public string Cedula { get; set; } = "";
        public string Nombres { get; set; } = "";
        public string? Email { get; set; }
        public string? Telefono { get; set; }
        public string? Direccion { get; set; }
        public bool Estado { get; set; } = true;
        public DateTime Creado_En { get; set; }
    }
}
